view-admin/src/pages/index.tsx

```diff
@@ -138,9 +138,17 @@ const Page: NextPage = () => {
                   </div>
                 )}
               </div>
-              <button type="submit" className={styles.Button}>
-                送信
-              </button>
+
+              {!errors.submitNumber && (
+                  <button type="submit" className={styles.Button}>
+                    送信
+                  </button>
+              )}
+              {errors.submitNumber && (
+                    <button type="submit" className={styles.dummyButton}>
+                      送信
+                    </button>
+              )}
```

Comment:
```suggestion
              <button
                type="submit"
                className={
                  errors.submitNumber ? styles.dummyButton : styles.Button
                }
              >送信</button>
```
こんな感じでclassNameの中で書けて、コード量短くできるよ～